wapi/util.py

```diff
@@ -182,37 +186,37 @@ def parsetime(datestr, tz=None):
     """
     Parse the input date and optionally convert to correct time zone
     """
+
     d = dateutil.parser.parse(datestr)
+
     if tz is not None:
-        # Convert timestamp to given tz
-        if isinstance(tz, str):
+        if not isinstance(tz, tzinfo):
+            # Raises `pytz.exception.UnknownTimeZoneError`
+            # if `tz` is not the name of a timezone
             tz = pytz.timezone(tz)
-        try:
+
+        if d.tzinfo:
```

Comment:
Jeg er som vanlig skeptisk til truthy/falsy-testing med mindre det faktisk må brukes.  Fungerer det med `is not None` her?